tests/unit/ReportUtilsTest.ts

```diff
@@ -3,14 +3,42 @@ import {addDays, format as formatDate} from 'date-fns';
 import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
 import Onyx from 'react-native-onyx';
 import DateUtils from '@libs/DateUtils';
-import * as ReportUtils from '@libs/ReportUtils';
-import * as TransactionUtils from '@libs/TransactionUtils';
+import {
+    buildOptimisticCreatedReportAction,
+    buildOptimisticExpenseReport,
+    buildOptimisticIOUReportAction,
+    buildParticipantsFromAccountIDs,
+    buildTransactionThread,
+    getAllAncestorReportActions,
+    getApprovalChain,
+    getChatByParticipants,
+    getDefaultWorkspaceAvatar,
+    getDisplayNamesWithTooltips,
+    getGroupChatName,
+    getIconsForParticipants,
+    getInvoiceChatByParticipants,
+    getMostRecentlyVisitedReport,
+    getQuickActionDetails,
+    getReportIDFromLink,
+    getReportName,
+    getWorkspaceIcon,
+    getWorkspaceNameUpdatedMessage,
+    isChatUsedForOnboarding,
+    requiresAttentionFromCurrentUser,
+    shouldDisableThread,
+    shouldReportBeInOptionList,
+    temporary_getMoneyRequestOptions,
+} from '@libs/ReportUtils';
```

Comment:
Is this a new thing we're trying to do everywhere - import specific functions instead of `import * as <something>`? Or is it just your personal preference?